aiomisc/entrypoint.py

```diff
@@ -64,6 +71,25 @@ def __init__(self, *services, loop: asyncio.AbstractEventLoop = None,
         self.services = services
         self.shutting_down = False
 
+        self.post_stop = Signal()
+        self.post_stop.connect(self.clear_dependencies)
```

Comment:
I think this code should be place in the dependencies module. It’s will proof of the signals will working on user code. Dog fooding, but excellent proof.